core/sys/darwin/Security/SecRandom.odin

```diff
@@ -0,0 +1,19 @@
+package Security
+
+import CF "core:sys/darwin/CoreFoundation"
+
+foreign import Security "system:Security.framework"
+
+// A reference to a random number generator.
+RandomRef :: distinct rawptr
+
+@(link_prefix="Sec", default_calling_convention="c")
+foreign Security {
+	// Default random ref for /dev/random. Synonym for nil.
+	@(link_name="kSecRandomDefault") kSecRandomDefault: RandomRef
```

Comment:
Why is the `link_name` set if the it has the exact same name?

Reply:
Because `foreign Security {` has `@(link_prefix="Sec"` on it.
Wouldn't it try to import as `SeckSecRandomDefault` without the `@(link_name="kSecRandomDefault")`?

Reply:
just like `RandomCopyBytes` and `CopyErrorMessageString` below that are actually `SecRandomCopyBytes` and `SecCopyErrorMessageString`